test/utils.ts

```diff
@@ -37,6 +38,8 @@ const initializeContracts = async function (
 
   const minimumStakeProportion = opts.minimumStakeProportion ?? 2000;
 
+  const oracle = opts.oracle ?? deployer;
```

Comment:
Should we name `seekerPowerOracleAddr` to be clear? For both constant and options's name.


Or can we just do as below in line 129?
```suggestion
  await seekerPowerOracle.initialize(opts.oracle ?? deployer, { from: deployer });
```

Reply:
I put the `const oracle` to keep in line with how we defined the other default options. I think we can renames this to `seekerPowerOracle` just to be clearer.